isis/scripts/install_isis.sh

```diff
@@ -0,0 +1,355 @@
+        ENV_NAME=$isis_env_name
+    fi
+fi
+
+# Check if the environment already exists 
+if conda env list | grep -qE "^$ENV_NAME[ ]."; then 
+    printf "\nEnvironment \"$ENV_NAME\" already exists. Not performing any updates.\n" 
+    printf "To delete the old environment, use the following commands:\n\n" 
+    printf "\t$ conda deactivate\n" 
+    printf "\t$ conda remove -n $ENV_NAME --all\n\n" 
+else
+    conda config --set ssl_verify false 
+
+    # Create a new environment with the specified package
+    echo "Creating a new environment: $ENV_NAME and installing $PACKAGE_NAME"
+    mamba create -c conda-forge -c usgs-astrogeology -n $ENV_NAME $PACKAGE_NAME=$ISIS_VERSION $LIBGL_INSTALL rclone -y || {
+        echo "Failed to install $PACKAGE_NAME=$ISIS_VERSION"
+        echo "Seaching for $PACKAGE_NAME versions ..."
+        mamba search -c conda-forge -c usgs-astrogeology $PACKAGE_NAME || failed_command "Search for $PACKAGE_NAME versions"
+        exit 1
+    }
+    echo "Environment \"$ENV_NAME\" created and $PACKAGE_NAME installed."
+fi
+
+# If a ISISDATA_PREFIX was not set, ask the user for it
+if [ -z "$ISISDATA_PREFIX" ]; then
+    ISISDATA_PREFIX="$HOME/.Isis/isis_data"
+    printf "ISISDATA directory is needed for most applications and is several Gbs in size. \n"
+    printf "By default, ISISDATA will be installed at this location:\n\n"
+    printf "\t$ISISDATA_PREFIX\n"
+    printf "\\n"
+    printf "  - Press ENTER to confirm the ISISDATA location\\n"
+    printf "  - Press CTRL-C to abort the installation\\n"
+    printf "  - Or specify a different location below\\n"
+    printf "\\n"
+    printf "[%s] >>> " "$ISISDATA_PREFIX"
+
+    read -r isis_data_path
+
+    # If no input, use default path
+    if [ -n "$isis_data_path" ]; then
+        ISISDATA_PREFIX=$isis_data_path
+    fi
+
+    echo "Setting ISISDATA to $ISISDATA_PREFIX"
+fi
+
+# Verify if the folder exists
+if ! [ -d "$ISISDATA_PREFIX" ]; then
+    if [ "$DOWNLOAD_DATA" = "YES" ]; then
+        echo "Creating folder $ISISDATA_PREFIX"
+        mkdir -p $ISISDATA_PREFIX || failed_command "Creating $ISISDATA_PREFIX"
+    else
+        echo "Not creating directory $ISISDATA_PREFIX since the --no_data flag was set"
+    fi
+fi
+
+conda env config vars set -n $ENV_NAME ISISDATA=$ISISDATA_PREFIX ISISROOT=$MINIFORGE_DIR/envs/$ENV_NAME || failed_command "Conda config var set"
+
+if [[ "$MINIFORGE_DIR" == *"envs/$ENV_NAME"* ]]; then
+    ENV_PATH="$MINIFORGE_DIR"
+else
+    ENV_PATH="$MINIFORGE_DIR/envs/$ENV_NAME"
+fi
+
+DOWNLOAD_ISIS_DATA_SCRIPT="$ENV_PATH/bin/downloadIsisData"
+if [[ ! -f "$ENV_PATH/bin/downloadIsisData" && ! -f "$ENV_PATH/etc/isis/rclone.conf" ]]; then
+    if ! [ -f "$DOWNLOAD_ISIS_DATA_SCRIPT" ]; then
+        printf "\nInstalled download script: $DOWNLOAD_ISIS_DATA_SCRIPT\n"
+        curl  --output "$ENV_PATH/bin/downloadIsisData" -LJO --insecure https://github.com/USGS-Astrogeology/ISIS3/raw/dev/isis/scripts/downloadIsisData
```

Comment:
running `curl --insecure` is bad security practice. Is this really necessary here?

Reply:
Likely a cert issue on my end